config/global.ini.php

```diff
@@ -761,6 +761,15 @@
 ; Note: setting this to 0 increases your users' privacy.
 enable_fingerprinting_across_websites = 0
 
+; When enabled (set to 1) and cookies are disabled in the tracker, then the fingerprint will be restricted even further
+; to prevent a visitor to be tracked over multiple days. This means typically that no cookie consent is needed when cookies
+; are disabled. You may disable this feature if you don't want to use cookies in the browser, but still be able to get more
```

Comment:
> You may disable this feature if you don't want to use cookies in the browser, but still be able to get more accurate unique visitor numbers.

Actually afaik, you would disable this feature only if you have set a `window_look_back_for_visitor` value higher than 24 hours. By default window_look_back_for_visitor = 0 which means we only look back 30min to match a visit. Returning visitors wouldn't even be matched from 2 hours ago. So it's only when cookies are disabled and window_look_back_for_visitor > say 43200 then it might be useful to deactivate this.

Reply:
Are you certain fingerprint is only looked back 30 minutes by default? Then I'd remove the config setting, and also the extra tracking parameter `cd` when cookies are disabled and we simply do apply the shorter fingerprint always unless `window_look_back_for_visitor > 24 hours` (but even then it would be only needed probably when not a new visit should be created after midnight). So we could apply this more privacy friendly fingerprint pretty much always

Reply:
looking at the code, yes it looks like the window_look_back_for_visitor is only used for fingerprint and defaults to 30min (defaults to the visit duration setting)